server/eclipse-project/src/main/java/us/freeandfair/corla/query/ExportQueries.java

```diff
@@ -269,4 +284,63 @@ public static long custCopyOut(final String sql, OutputStream to, CopyManager cm
     }
   }
 
```

Comment:
In the first sentence of the comment, clarify that the ContestResult structure does not have the correct values for those quantities *for IRV*.